equations.md

```diff
@@ -29,7 +29,7 @@ new bonds. This creates a secondary price discovery mechanism that is somewhat
 independent from the actual market.
 
 $$
-Bond Price = RFV / Premium
+Bond Price = RFV * Premium
```

Comment:
Can you explain that change. I honestly do not follow how these equations came to be and why they are right or wrong. It would help my understanding to hear a reason for this particular change.

Reply:
I referred to this article: https://olympusdao.medium.com/a-primer-on-oly-bonds-9763f125c124. Under the section "Why Do I Want to Bond?", you can find the graph for different prices when the bonding premium is 2. But in that article, the formula listed is clearly wrong: Executing Price = RFV / Premium {Premium ≥ 1}. Because if premium is always larger than 1, Executing Price (or bond price) will always be smaller than RFV, which can't be true.

Reply:
Sorry I do not follow the whole thing at all. Like my brain is not able to comprehend all the assumptions made here. Somebody else better review this PR.

Reply:
I didn't make any assumption here; it is based on the graph given in the article

Reply:
What I tried to say is, is my brain does not even math. These equations rarely tell me anything. I could ask all kinds of questions how this thing is meant to express anything but somebody better review this thing.

Reply:
After checking with Zeus and unbanksy, the equations under the bond section have been updated. See if they make sense now.